client/src/common/constants/propTypes.js

```diff
@@ -41,3 +41,13 @@ export const ResourcesDataPropType = PropTypes.shape({
   cohorts: ResourcePropType,
   lists: ResourcePropType
 });
+
+export const licensePropType = PropTypes.shape({
+  licenseText: PropTypes.string,
+  licenseType: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.arrayOf(PropTypes.string)
+  ]),
+  licenseUrl: PropTypes.string,
+  repository: PropTypes.string
```

Comment:
it is already `licensePropType` so maybe remove `license` prefix for `text`, `type` and `url`?